src/com/jalasoft/search_files/utils/Search_Validation.py

```diff
@@ -47,7 +47,7 @@ def is_less_than_or_equal(self, bit_size, mega_bit_size):
     def is_bigger_than_other(self, bit_size, mega_bit_size):
         # it is possible that the is_bigger_than_other() method is redundant
         num_bit = bitmath.Bit(bit_size)
-        num_mega_bit = bitmath.Mb(mega_bit_size)
+        num_mega_bit = bitmath.Mib(mega_bit_size)
```

Comment:
Please, see the following page were there is an exmplete how convert to x computer capacite: https://pypi.python.org/pypi/bitmath
Convert Units
File size unit conversion:

>>> from bitmath import *
>>> dvd_size = GiB(4.7)
>>> print "DVD Size in MiB: %s" % **dvd_size.to_MiB()**
DVD Size in MiB: 4812.8 MiB